spec/beaker/host/cisco_spec.rb

```diff
@@ -14,10 +14,10 @@ module Cisco
 
     describe '#prepend_commands' do
 
-      context 'for cisco-5' do
+      context 'for cisco-nexus' do
 
         before :each do
-          @platform = 'cisco-5-x86'
+          @platform = 'cisco-nexus-x86'
```

Comment:
This seems wrong still - we still need the ability to specify an OS version in case there are behavior differences between, say, NX-OS 7.0 and NX-OS 7.1. Rather than treating `nexus` or `ios_xr` as the 'version', they should be part of the 'os' - so something more like `cisco_nexus-7.0(3)I2(1)-x86_64` or `cisco_ios_xr-6.0.0-x86_64` is what I would expect Beaker to accept.